src/wasm/main.cpp

```diff
@@ -0,0 +1,41 @@
+#include "tree_parse.hpp"
+
+int main() {
+    std::ifstream file("data2.json"); //this is only for test
+    json j;
+    file >> j;
+
+    u_int16_t start_time = 0;
+
+    if(j.contains("first_time")){
+         if (j["first_time"].is_number_unsigned()) {
+            start_time = j["first_time"].get<uint64_t>();
+        } else if (j["first_time"].is_number_integer()) {
+            start_time = static_cast<uint64_t>(j["first_time"].get<int64_t>());
+        }
```

Comment:
Unless a compiler complains, you can safely assume that ```j["first_time"]``` is an unsigned integer, as a timestamp cannot be a negative number.

Reply:
Done, seems to be working fine. :+1: